llvm/include/llvm/ObjCopy/CommonConfig.h

```diff
@@ -218,6 +218,7 @@ struct CommonConfig {
   uint64_t PadTo = 0;
   StringRef SplitDWO;
   StringRef SymbolsPrefix;
+  StringRef SymbolsPrefixRemove;
```

Comment:
Please make sure that non-ELF formats complain when this option is used, in the same manner as `SymbolsPrefix`. See `ConfigManager.cpp` for this.